internal/controller/capacity.go

```diff
@@ -68,8 +68,8 @@ func (r *InstasliceReconciler) findNodeAndDeviceForASlice(ctx context.Context, i
 			}
 			size, discoveredGiprofile, Ciprofileid, Ciengprofileid := r.extractGpuProfile(instaslice, profileName)
 			resourceIdentifier := pod.Spec.Containers[0].EnvFrom[0].ConfigMapRef.Name
-			allocDetails := policy.SetAllocationDetails(profileName, uint32(newStart), uint32(size),
-				string(pod.UID), instaslice.Name, "creating", discoveredGiprofile,
+			allocDetails := policy.SetAllocationDetails(profileName, newStart, uint32(size),
+				string(pod.UID), instaslice.Name, string(inferencev1alpha1.AllocationStatusCreated), discoveredGiprofile,
```

Comment:
Although replacing with a constant makes sense, could you please double-check if changing from `...Creating` to `...Created` is correct?

Reply:
yea that is a bug. good find.